lib/simp/rake/helpers/assets/rpm_spec/simp6.spec

```diff
@@ -287,6 +380,9 @@ curdir=`pwd`
 dirname=`basename $curdir`
 cp -r ../$dirname %{buildroot}/%{prefix}/%{module_name}
```

Comment:
why not use cp -L instead of removing links?  That allows the capability to still work.